controllers/configurationpolicy_controller.go

```diff
@@ -2653,6 +2644,8 @@ func (r *ConfigurationPolicyReconciler) checkAndUpdateResource(
 
 				if !isInform {
 					log.Info("Queuing an update for the object due to a value mismatch", "key", key)
+				} else {
+					log.Info("Detected value mismatch for object key: " + key)
```

Comment:
This seemed like a nice little add to the logging since we're already doing it for `enforce`, but I can take it out (or move it to level 2) if we think it'd be too noisy.